test/test_crypten.py

```diff
@@ -302,9 +298,8 @@ def forward(self, input):
         out = self.nested(out)
 
     def set_all_parameters(self, value):
```

Comment:
Q: Is there any plan to have a method like this added directly in PyTorch for  ```nn.Module```?

Reply:
I don't believe there is any plan to do this. This function isn't defined in CrypTen either - it is defined in a unit test as a helper function. In general, I think it is left to the user to initialize (or otherwise set) parameter values.